tools/klee/main.cpp

```diff
@@ -508,8 +509,20 @@ void KleeHandler::processTestCase(const ExecutionState &state,
     if (errorMessage || WriteKQueries) {
       std::string constraints;
       m_interpreter->getConstraintLog(state, constraints,Interpreter::KQUERY);
+      ConstraintManager constraintsWithError =
+          state.symbolicError->getConstraintsWithError();
       llvm::raw_ostream *f = openTestFile("kquery", id);
       *f << constraints;
+      *f << "\nPath conditions with error:";
```

Comment:
@Himeshi You may want to ensure that this is output only when --precision is specified. Also, perhaps it's best to output to a separate file, maybe a .kquery_precision_error file for easier processing by a script, but I have no string opinion on this.